### PR TITLE
To Mac: enhance light representation & modified light toggle event

### PR DESCRIPTION
**Pull request by zhna123**
[#9](https://github.com/champgm/hue-stuff/issues/9)

### CODE REVIEW

File: hue-web/src/main/java/com/cgm/java/hue/web/ToggleLight.java [hunk 1]
```diff
@@ -38,11 +37,6 @@ protected void doGet(final HttpServletRequest request, final HttpServletResponse
         LOGGER.info("Attempting to toggle the light with ID: " + lightId);
         final Light toggledLight = LIGHT_UTIL.toggleLight(HUE_CONFIGURATION.getIP(), HUE_CONFIGURATION.getToken(), lightId);
         LOGGER.info("State successfully toggled to ON = " + toggledLight.getState().getOn());
-
-        // Return the result
```
Comment (champgm) [inline hue-web/src/main/java/com/cgm/java/hue/web/ToggleLight.java]: Why did you remove this? Not necessary anymore?

Comment (zhna123) [inline hue-web/src/main/java/com/cgm/java/hue/web/ToggleLight.java]: Yes I never actually needed it. It is not necessary to return the toggled light because it is still the original light. After being toggled, it will switch to the new state.
